lib/ArangoDBClient/Endpoint.php

```diff
@@ -55,12 +55,12 @@ class Endpoint
     /**
      * Regexp for TCP endpoints
      */
-    const REGEXP_TCP = '/^tcp:\/\/(.+?):(\d+)\/?$/';
+    const REGEXP_TCP = '/^(tcp|http):\/\/(.+?):(\d+)\/?$/';
 
     /**
      * Regexp for SSL endpoints
      */
-    const REGEXP_SSL = '/^ssl:\/\/(.+?):(\d+)\/?$/';
+    const REGEXP_SSL = '/^(ssl|https):\/\/(.+?):(\d+)\/?$/';
```

Comment:
You can also use a different delimiter than `/` to avoid escaping the slashes, e.g. `'#^(ssl|https)://(.+?):(\d+)/?$#'`;